packages/examples/background_agent/demo-background-agent.ts

```diff
@@ -0,0 +1,217 @@
+/**
+ * @license
+ * Copyright 2025 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { McpServer } from '@modelcontextprotocol/sdk/server/mcp.js';
+import { StdioServerTransport } from '@modelcontextprotocol/sdk/server/stdio.js';
+import { z } from 'zod';
+
+const BackgroundAgentMessageSchema = z.object({
```

Comment:
We should b e exposing all of this schema so that consumers can reference a library and/or we should also be leaning on `@google/genai` types